src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * A task list handler that adds/deletes tasks in the list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public int getCount() {
+        return tasks.size();
+    }
+
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
```

Comment:
I'm slightly concern about the public `getTasks()` method as anyone can access this and they can modify your array. Alternatively, I was thinking this method can be a `getTask(int index)` method that returns `Task` at a specific index instead of `ArrayList<Task>`. In the rest of the files, the task list can be passed in as parameter and in the body you can `getTask(int index)` and then continue with the logic.